api/stse_device_authentication.h

```diff
@@ -35,51 +35,52 @@
 /**
  * \brief Get STSE Device ID
```

Comment:
Please align this header file with comments in source file for function headers and theirs doxygen documentation